### PR TITLE
set up a webpack dev enviroment

### PR DESCRIPTION
**Pull request by Kareem24**
chore: add navbar and the footer section
feat: Add navbar onClick funtionalities so as to change the pages on click without re-loading the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,22 +3,86 @@
...
+    </div>
+    <div class="date section-center">
+      <p class="recent-date">
+        <!-- <span class="day">Monday</span>
+        <span class="date">19</span>
+        <span class="year">2023</span>
+        <span class="hour">8</span>
+        :
+        <span class="hour">00</span> -->
+        <!-- <span class="meridian">am</span> -->
```
Comment (Lanr3waju) [inline index.html]: Remove this and other comments

File: mightuse.md
Comment (Lanr3waju) [inline mightuse.md:1]: what's this file for and why the name"mightuse", should be something descriptive

File: src/css/main.css [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+:root {
+  --black: #002;
+  --white: #fff;
+  --light-gray: #d3d3d3;
+  --dark-gray: #f3f0f0;
+}
+
+html {
+  overflow-x: hidden;
```
Comment (Lanr3waju) [inline src/css/main.css:11]: Try to fix your `footer` issue and don't use this to conceal it.

File: src/css/main.css [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+}
+
+.logo {
+  font-weight: 700;
+  color: var(--black);
+}
+
+.navbar {
+  justify-content: space-between;
+  width: 90%;
```
Comment (Lanr3waju) [inline src/css/main.css]: I think this width should be set to 100% and padding should be added to the parent element to achieve what you want to achieve here.

File: src/index.html
Comment (Lanr3waju) [inline src/index.html:1]: why do you have two `index.html` files?

**Review by Lanr3waju — COMMENTED** [review]
Great Job!

File: src/css/main.css (hunk 2 above)
Comment (Kareem24) [reply src/css/main.css:11]: the 
```
overflow:hidden
``` 
on the html element, It's used to hide my mobile menu

Cause I use relative position on my body element and absolute to my nav links and it's giving it a horizontal scroll

**Review by Lanr3waju — APPROVED** [review]
LGTM!!

**Pull request merged**